low-level/sensors/impl/generic/raid.py

```diff
@@ -282,7 +324,7 @@ def _process_mdstat(self):
             # Parse out status' and path info for each drive
             if "md" in fields[0]:
                 self._device = f"/dev/{fields[0]}"
-                self._devices.append(self._device)
+                self._devices.add(self._device)
```

Comment:
we need to maintain order of devices as it appear in conf file. why change here?

Reply:
the same device used to get added in every iteration. set is efficient in search that's why I used but if we need to maintain order then I can just check once if a device is present in the list or not before adding.

Reply:
no checking if device would not main order from conf file. we just need to clear _devices list during each iteration then?
is that okay?

Reply:
cleared _devices list in each iteration and kept is as list preserving order from /proc/mdstat file.

Reply:
understood, thanks